### PR TITLE
CC-SP4-101.8.3 updated cab loaction API

### PR DESCRIPTION
**Pull request by ankit-mogha**
What does this change do?
Added API for updating cab location

Any additional information for the reviewer to start
NA

How should this be manually tested?
We need window OS in which Rust is installed. then execute this rust program

Which Trello task and sub-task does this change relate to?
CC -SP4- 101.8.3 - Polkadot API for update cab location.

Are there any changes pending?
No

Does any team have to be notified of changes in this feature?
Yes

Definition of Done:
- [ ] Is there >90% unit test code coverage?
- [ ] Does this PR add new dependencies? If so, please list out the same.
- [ ]  Will this feature require a new piece of infrastructure to be implemented?
- [x] Is there appropriate logging included?
- [x] Does the project compile ok?
- [x] Have Clippy violations been fixed?
- [x] Have Code is properly formatted?

### CODE REVIEW

File: app/src/index.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        const alice = keyring.addFromUri('//Alice');
+        const types = JSON.parse(readFileSync('./types.json', 'utf8'));
+        const api = await ApiPromise.create({ provider: wsProvider,
+            types
+        });
+
+        try{
+            let namePromise = digestMessage("Ankit Mogha");
+            namePromise.then((cust)=>{
+                const updateCustLocation = api.tx.carpooling.updateCustomerLocation(15,{id: 15, name: "0x"+cust, location: [20,40]});
```
Comment (aman2909verma) [inline app/src/index.js:48]: wrong parameters for the function

**Review by aman2909verma — CHANGES_REQUESTED** [review]
do not push the idea folder and correct the code

File: app/src/index.js (hunk 1 above)
Comment (aman2909verma) [inline app/src/index.js:48]: wrong function called. call function to update cab location, not the customer.

**Review by aman2909verma — CHANGES_REQUESTED** [review]

File: app/src/index.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        const alice = keyring.addFromUri('//Alice');
+        const types = JSON.parse(readFileSync('./types.json', 'utf8'));
+        const api = await ApiPromise.create({ provider: wsProvider,
+            types
+        });
+
+        try{
+            let namePromise = digestMessage("Ankit Mogha");
+            namePromise.then((cust)=>{
+                const updateCabLocation = api.tx.carpooling.updateCabLocation(15,{id: 15, location: [20,40]});
```
Comment (aman2909verma) [inline app/src/index.js:48]: wrong parameters passed for the function

File: app/src/index.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        const wsProvider = new WsProvider('ws://127.0.0.1:9944');
+        const keyring = new Keyring({ type: 'sr25519' });
+        const alice = keyring.addFromUri('//Alice');
+        const types = JSON.parse(readFileSync('./types.json', 'utf8'));
+        const api = await ApiPromise.create({ provider: wsProvider,
+            types
+        });
+
+        try{
+            let namePromise = digestMessage("Ankit Mogha");
```
Comment (aman2909verma) [inline app/src/index.js:46]: digestMessage function not needed

**Review by aman2909verma — CHANGES_REQUESTED** [review]

**Comment by aman2909verma** [comment]
do not push idea folder

**Review by Ayushi-knoldus — COMMENTED** [review]
remove extra files please

**Review by aman2909verma — APPROVED** [review]

**Pull request closed without merging**